pyproject.toml

```diff
@@ -32,7 +32,6 @@ dependencies = [
   "azure-storage-file-datalake~=12.10",
   "azure-storage-file-share~=12.10",
   "chevron~=0.14",
-  "chili~=2.1",
```

Comment:
Did we replace chili with pydantic?

Reply:
Yes, I think this must have snuck back in through a merge somewhere.